.github/workflows/llm_example_tests.yml

```diff
@@ -0,0 +1,80 @@
+name: LLM Example Test
+
+# Cancel previous runs in the PR when you push new commits
+concurrency:
+  group: ${{ github.workflow }}-llm-example-tests-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+# Controls when the action will run. 
+on:
```

Comment:
Should we trigger the example test nightly? Maybe there is no need to trigger it every PR. The example test will be heavy later.
Refer to the "on" condition here: https://github.com/intel-analytics/BigDL/blob/main/.github/workflows/llm-nightly-test.yml#L9

Reply:
BTW, should we just put it into the nightly test yml?

Reply:
I suppose this should be first run in the PR to make sure it succeeds before the merge. Then we can move to nightly only.

Reply:
https://github.com/intel-analytics/BigDL/blob/main/.github/workflows/llm-nightly-test.yml#L9 The on condition here will trigger the test for the PR related to the example yml, and trigger this test nightly after merging this PR.